homeassistant/components/recorder/db_schema.py

```diff
@@ -572,10 +571,12 @@ def shared_attrs_bytes_from_event(
         exclude_attrs = set(ALL_DOMAIN_EXCLUDE_ATTRS)
         if base_platform_attrs := exclude_attrs_by_domain.get(domain):
             exclude_attrs |= base_platform_attrs
```

Comment:
This can be removed when all integrations with recorder platforms have been migrated